apps/condo/domains/resident/schema/FindOrganizationsByAddressService.js

```diff
@@ -14,7 +14,7 @@ const {
     getOrganizationsWithAcquiring,
     getOrganizationsWithMeters,
     getBillingInformationForOrganizations,
-} = require('@condo/domains/resident/utils/serverSchema/findOrganizationsForAddress')
+} = require('@condo/domains/resident/utils/serverSchema/FindOrganizationsByAddress')
```

Comment:
Please use case-sensitive mode for search-n-replace :)
```suggestion
} = require('@condo/domains/resident/utils/serverSchema/findOrganizationsByAddress')
```

Reply:
Fixed it ty